### ISSUE TITLE
Help me pls

### ISSUE THREAD

**Issue by EvvaLassed** [dalaranwow/dalaran-wow#5966]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. hello guys. my name is Evva and i am new at wow 
2. ive played 1 month and ive quited retail to play woltk Because of my friends rec
3. i didnt got the lvl 70 instant boost, someone can help me with that ? 
4. thank u >3
3. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
Greetings @EvvaLassed 
This is the in-game bugtracker, which lists only in-game issues that can affect players, and not the website features. Please make an in-game ticket or send us an e-mail to "dalaran.community@gmail.com" and we will help you out as soon as possible. Make sure you give us your account name, what character you used the boost and when, and any other relevant info you can provide.

Cheers,
Golgy
